includes/widgets/class-amp-widget-archives.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Class AMP_Widget_Archives
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Widget_Archives
+ *
+ * @package AMP
+ */
+class AMP_Widget_Archives extends WP_Widget_Archives {
+
+	/**
+	 * AMP_Widget_Archives constructor.
+	 *
+	 * @return void.
+	 */
+	public function __construct() {
+		parent::__construct();
+		add_filter( 'amp_component_scripts', array( $this, 'form_script' ) );
+	}
+
+	/**
+	 * Add a form extension script for AMP to output.
+	 *
+	 * @param array $scripts The AMP scripts.
+	 * @return array $scripts The filtered AMP scripts.
+	 */
+	public function form_script( $scripts ) {
+		if ( ! isset( $scripts['amp-form'] ) ) {
+			$scripts['amp-form'] = 'https://cdn.ampproject.org/v0/amp-form-latest.js';
```

Comment:
This is unnecessary once you merged from `develop` in 62a924ff761b0a24985b4974d97aa77ffda466b7